security/src/test/scala/pl/touk/nussknacker/ui/security/oauth2/OAuth2AuthenticationResourcesSpec.scala

```diff
@@ -70,11 +71,20 @@ class OAuth2AuthenticationResourcesSpec extends FunSpec with Matchers with Scala
   }
 
   it("should redirect for good authorization token") {
-    authenticationOauth2(authenticationResources, "B5FwrdqF9cLxwdhL") ~> check {
+    authenticationOauth2(authenticationResources(), "B5FwrdqF9cLxwdhL") ~> check {
       status shouldBe StatusCodes.OK
+      header[`Set-Cookie`] shouldBe None
       val response = responseAs[Oauth2AuthenticationResponse]
       response.accessToken shouldEqual "AH4k6h6KuYaLGfTCdbPayK8HzfM4atZm"
       response.tokenType shouldEqual "Bearer"
     }
   }
+
+  it("should set cookie in response if configured") {
+    val cookieConfig = TokenCookieConfig("customCookie", Some("/myPath"), None)
+    authenticationOauth2(authenticationResources(config = defaultConfig.copy(tokenCookie = Some(cookieConfig))), "B5FwrdqF9cLxwdhL") ~> check {
+      status shouldBe StatusCodes.OK
+      header[`Set-Cookie`] shouldBe Some(`Set-Cookie`(HttpCookie(name = cookieConfig.name, value = "AH4k6h6KuYaLGfTCdbPayK8HzfM4atZm", httpOnly = true, path = cookieConfig.path)))
```

Comment:
this value is related with "B5FwrdqF9cLxwdhL"?  Can u show this relation?

Reply:
:+1: I extracted variables...